documentation/providers/luadns.md

```diff
@@ -0,0 +1,38 @@
+## Configuration
+
+To use this provider, add an entry to `creds.json` with `TYPE` set to `LUADNS`
+along with your [email and API key](https://www.luadns.com/api.html#authentication).
+
+Example:
+
+```json
```

Comment:
Would you mind applying my previous change from https://github.com/StackExchange/dnscontrol/pull/2119 to `creds.json` and `dnsconfig.js` as well? 🙏